buildenv/jenkins/variables/defaults.yml

```diff
@@ -193,9 +193,9 @@ s390x_linux:
     8: 'linux-s390x-normal-server-release'
     11: 'linux-s390x-normal-server-release'
   node_labels:
-    build: 'ci.role.build && hw.arch.s390x && sw.os.ubuntu'
+    build: 'ci.role.build && hw.arch.s390x && sw.os.rhel'
```

Comment:
Should probably make this specifically `sw.os.rhel.7`. In the event we add rhel8 for example.